README.md

```diff
@@ -1,41 +1,121 @@
 # bbb-configs (Ansible Config Management)
 
+## FAQ
+
 ### How to get started
 
-```
+```sh
 pip3 install -r requirements.txt
 ```
 
 ### How to spin up a config run (generate only, output path is /tmp/configs/..:
 
-```
+```sh
 ansible-playbook play.yml
 ```
 
 ### How to spin up a config run and generate images
 
-```
+```sh
 ansible-playbook play.yml --tags image
 ```
 
 ### How to limit image creation to a single location
 
-```
+```sh
 ansible-playbook play.yml --limit example-core
 ansible-playbook play.yml --limit example-*
 ```
 
 ### How to spin up a config run, generate image and flash (Requires IPv6 Connectivity from inside freifunk network, not yet fully working)
 
-```
+```sh
 ansible-playbook play.yml --tags flash
 ```
 
 ### Whats required to bringup a location?
 
-```
+```sh
 /group_vars : Create location folder
 /host_vars : Create host folder for every openwrt device
 Done!
 
 ```
+
+## Variables and structures explained
+
+### group_vars/
+
+`group_vars/` holds tree types of resources: the `all/`-directory, location directories and model files.
+
+#### `all/`
+
+This directory holds base configuration options that will be inserted into every router configuration. Each device will get this options, like timezones, DNS-servers, the packagefeed-URL and so on.
+
+There are also files for the standard ssh keys and definitions for the wifi profiles.
+
+#### model_files
+
+These files define how bbb-configs needs to handle different hardware models. This example shows a WDR4900:
+```sh
+---
+imagebuilder: "https://downloads.openwrt.org/releases/21.02-SNAPSHOT/targets/mpc85xx/p1010/openwrt-imagebuilder-21.02-SNAPSHOT-mpc85xx-p1010.Linux-x86_64.tar.xz"
+feed_version: "1.2.0-snapshot"
+imagebuilder_disable_signature_check: true      # don't check the signature-keys when installing packages ??
+
+switch_ports: 6             # What's that for?
+switch_int_port: 0          # What's that for?
+switch_ignore_ports: []     # What's that for?
```

Comment:
sometimes, not all switch ports are usuable. for example the sxtsq 5 ac has "6" ports, but oly cpu port + port 5 is used (1 port device)